src/os/FileStore.cc

```diff
@@ -2011,7 +2014,10 @@ int FileStore::queue_transactions(Sequencer *posr, list<Transaction*> &tls,
 
   //prepare and encode transactions data out of lock
   bufferlist tbl;
-  int data_align = _op_journal_transactions_prepare(tls, tbl);
+  int orig_len = -1;
+  if (journal && journal->is_writeable()) {
```

Comment:
Because `if (journal && journal->is_writeable())` return false in _op_journal_transactions , 
if encode encode transactions data is not need.